modules/asr/intent_engine/intent_engine_io.c

```diff
@@ -96,6 +96,12 @@ void intent_engine_process_asr_result(int word_id)
         }
     }
     rtos_printf("RECOGNIZED: 0x%x, %s\n", (int) word_id, (char*)text);
+#if appconfINTENT_UART_CMD_INFO
+    static char res_info[128];
+    snprintf(res_info, 128-1, "Cmd:%s\r\n", text);
```

Comment:
It would be slightly better to use the `sizeof` operator to obtain the value `128-1`.